models/review.go

```diff
@@ -255,3 +255,22 @@ func UpdateReview(r *Review) error {
 	}
 	return nil
 }
+
+// PullReviewersWithType represents the type used to display a review overview
+type PullReviewersWithType struct {
+	User              `xorm:"extends"`
+	Type              ReviewType
+	ReviewUpdatedUnix util.TimeStamp `xorm:"review_updated_unix"`
+}
+
+// GetReviewersByPullID gets all reviewers for a pull request with the statuses
+func GetReviewersByPullID(pullID int64) (issueReviewers []*PullReviewersWithType, err error) {
+	err = x.Select("`user`.*, review.type, max(review.updated_unix) as review_updated_unix").
+		Table("review").
+		Join("INNER", "user", "review.reviewer_id = `user`.id").
+		Where("review.issue_id = ? AND (review.type = ? OR review.type = ?)", pullID, ReviewTypeApprove, ReviewTypeReject).
+		GroupBy("`user`.id").
+		OrderBy("review.updated_unix").
```

Comment:
This will also most probably fail in most db types, should be `max(review.updated_unix)`

Reply:
I've changed it so it uses the value from `select`.